sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/CassandraResourcesOperationsTests.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class CassandraResourcesOperationsTests : CosmosDBManagementClientBase
+    {
+        public CassandraResourcesOperationsTests()
+            : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                InitializeClients();
+                resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                await CosmosDBTestUtilities.TryRegisterResourceGroupAsync(ResourceGroupsOperations, CosmosDBTestUtilities.Location, resourceGroupName);
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+           await CleanupResourceGroupsAsync();
+        }
+
+        [TestCase]
+        public async Task CassandraCRUDTestsAsync()
+        {
+            var cosmosDBClient = GetCosmosDBManagementClient();
```

Comment:
You already instantiated a cosmosDBClient as part of your class here: https://github.com/Azure/azure-sdk-for-net/blob/580a57495134e4c5fb40724f035c63c71ceba136/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementClientBase.cs#L43

Why are you making another client here?

Reply:
This is not resolved

Reply:
It is updated in the most recent commit, but I did not update in the comment.